packages/jest-cli/yarn.lock

```diff
@@ -1027,4 +1027,4 @@ yargs@~3.10.0:
     camelcase "^1.0.2"
     cliui "^2.1.0"
     decamelize "^1.0.0"
-    window-size "0.1.0"
+    window-size "0.1.0"
```

Comment:
Of course. The file was missing but that problem was fixed by https://github.com/facebook/jest/pull/3502